### PR TITLE
#278309: fixed stack overflow error when call parseint method with big strings…

### PR DESCRIPTION
**Pull request by rafael-santiago-take**
Adding unit test for identify stack overflow error when call parseint method with big strings on script process of blocks

### CODE REVIEW

**Review by EricJangola — APPROVED** [review]

File: src/Take.Blip.Builder.UnitTests/Actions/ExecuteScriptActionTests.cs [hunk 1]
```diff
@@ -299,5 +299,31 @@ public async Task ExecuteScripWithXmlHttpRequestShouldFail()
...
+                OutputVariable = nameof(result)
+            };
+            var target = GetTarget();
+
+            // Act
+            await target.ExecuteAsync(Context, JObject.FromObject(settings), CancellationToken);
+
+            // Assert
+            await Context.Received(1).SetVariableAsync(Arg.Any<string>(), Arg.Any<string>(), CancellationToken, Arg.Any<TimeSpan>());
+            await Context.Received(1).SetVariableAsync(nameof(result), result, CancellationToken, default(TimeSpan));
```
Comment (caiopsouza) [inline src/Take.Blip.Builder.UnitTests/Actions/ExecuteScriptActionTests.cs:326]: Are you asserting the result is NaN anywhere?

**Review by caiopsouza — APPROVED** [review]

File: src/Take.Blip.Builder.UnitTests/Actions/ExecuteScriptActionTests.cs (hunk 1 above)
Comment (rafael-santiago-take) [reply src/Take.Blip.Builder.UnitTests/Actions/ExecuteScriptActionTests.cs:326]: yes
